lib/Backends/Habana/Habana.cpp

```diff
@@ -1344,7 +1344,7 @@ bool HabanaBackend::isOpSupported(const NodeInfo &NI) const {
   case Kinded::Kind::AddNodeKind:
   case Kinded::Kind::AvgPoolNodeKind:
   case Kinded::Kind::BatchedAddNodeKind:
-  case Kinded::Kind::BatchedReduceAddNodeKind:
+  //case Kinded::Kind::BatchedReduceAddNodeKind:
```

Comment:
spurious?

Reply:
This was not intended to be part of the diff